### ISSUE TITLE
launch4j doesn't actually remove the need for Java?

### ISSUE THREAD

**Issue by Goober5000** [scp-fs2open/fsoinstaller#8]
It turns out that Launch4J doesn't actually remove the need for Java, as demonstrated by a recent Discord conversation:
![discord](https://user-images.githubusercontent.com/1878523/32126752-52d84f6e-bb40-11e7-9519-28a87bf7cf9a.png)

Is there any EXE generator which will generate a complete application with no Java dependencies?

Before you mention Microsoft JET, I already contacted Turey and he said his license is no longer valid.

**Comment by MageKing17** [comment]
You need to bundle a JRE with it. Here's the second result I got from my google search: https://stackoverflow.com/a/7582064/6407176

**Comment by asarium** [comment]
launch4j should show a dialog which contains a link to the java download page if Java isn't found on the system. Bundling a JRE is possible but that would increase the download size for only a tiny usability benefit.

**Comment by TheMatthew** [comment]
I would agree with asarium on that... Another issue would be the legality of bundling the jre as well as clashing with pre-installed versions.
